key_handling.c

```diff
@@ -38,6 +38,11 @@
 
 #include "key_handling.h"
 
+#define LOG_LOCAL_LEVEL ESP_LOG_DEBUG
+
+#define KEY_LIFETIME_IN_SECONDS (31536000) // one year
```

Comment:
include #ifndef and `compiler info message` and macro for calculating from years to seconds

Reply:
Instead of `#ifndef` the default config value is included in Kconfig.